steering_controllers_library/include/steering_controllers_library/steering_odometry.hpp

```diff
@@ -175,22 +175,23 @@ class SteeringOdometry
   /**
    * \brief Sets the wheel parameters: radius, separation and wheelbase
    */
-  void set_wheel_params(double wheel_radius, double wheelbase = 0.0, double wheel_track = 0.0);
+  void set_wheel_params(
+    const double wheel_radius, const double wheelbase = 0.0, const double wheel_track = 0.0);
```

Comment:
I personally don't like passing arguments by const copy 

It is better either const reference or just copy. As these are primitive data types, you can pass them without const. 

I believe you did it for consistency with other member methods?